src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,117 @@
+---
+protocol: "Uniswap-V3"
+website: "https://blog.uniswap.org/uniswap-v3"
+x: "https://x.com/uniswap"
+github: "https://github.com/Uniswap/v3-periphery/"
+defillama_slug: "uniswap-v3"
+chain: "Ethereum Mainnet"
```

Comment:
Ethereum instead of Ethereum Mainnet

this is for parsing and rendering necessary